conda_verify/utilities.py

```diff
@@ -1,6 +1,7 @@
 import re
 import sys
 
+import array
```

Comment:
Instead of using a bytearray, we should use the future library's bytes function: ``import future.builtins``